aide_design/unit_process_design/prefab/lfom_prefab_functional.py

```diff
@@ -37,157 +34,163 @@
 # The following constants need to go into the constants file
 Pi_LFOM_safety = 1.2
 # pipe schedule for LFOM
-SDR_LFOM = 26
-
+#SDR_LFOM = 26
+#FLOW = 10*u.L/u.s
+#HL_LFOM = 20*u.cm
+
+#primary outputs from this file are
+#Nominal diameter nom_diam_lfom_pipe(FLOW,HL_LFOM,Pi_LFOM_safety,SDR_LFOM)
+#number of rows n_lfom_rows(FLOW,HL_LFOM)
+#orifice diameter orifice_diameter(FLOW,HL_LFOM,drill_series_uom)
+#number of orifices in each row n_lfom_orifices(FLOW,HL_LFOM,drill_series_uom,SDR_LFOM)
+#height of the center of each row height_lfom_orifices(FLOW,HL_LFOM,drill_series_uom)
+
+# output is width per flow rate.
+@u.wraps(u.s/(u.m**2), [u.m,u.m], False)
 def width_stout(HL_LFOM,z):
-    return 2/((2*u.g_0*z)**(1/2)*math.pi*HL_LFOM)
-
-def n_lfom_rows(Q,HL_LFOM):
-    N_estimated = (HL_LFOM*math.pi/(2*width_stout(HL_LFOM,HL_LFOM)*Q)).to(u.dimensionless)
-    return min(10,max(4,math.trunc(N_estimated.magnitude)))
-
-FLOW = 5*u.L/u.s
-HL_LFOM = 20*u.cm
+    return (2/((2*pc.gravity*z)**(1/2)*ratio_VC_orifice*np.pi*HL_LFOM)).magnitude
+
+
+@u.wraps(None, [u.m**3/u.s,u.m], False)
+def n_lfom_rows(FLOW,HL_LFOM):
```

Comment:
capitalized parameters? They aren't constants... also, the width_stout needs a docstring comment